### DISCUSSION TITLE
Refer to front matter elsewhere

### DISCUSSION THREAD

**Discussion by Martien** [observablehq/framework#1798] [General]
I am converting some of my Wordpress blog posts to Framework:
1. Export Wordpress Into Xml File (WP > Tools > Export).
1. Generate markdown for each post using [wordpress-export-to-markdown](https://github.com/lonekorean/wordpress-export-to-markdown).
1. Post process each md file (clean up with sed and friends).

Given that:
- [Framework Markdown’ Front matter](https://observablehq.com/framework/markdown#front-matter) has a bunch of attributes. 
- The WP to MD step ads a couple besides `title`, like: `date`, `categories`, and `tags`.

Wish: use these attributes elsewhere and anywhere in subsequent md files. Especially the `date` value, as it is the posts’s publish date.

Alternative: in post processing (step 3 above), convert the date (if present) to a proper md element (e.g. a subtitle).

Thoughts?

Related to #1199

**Comment by mbostock** [comment]
What do you mean by “use these attributes elsewhere”? How are you going to use them?

I’d also consider using a [page loader](https://observablehq.com/framework/page-loaders) for this.

**Reply by Martien** [reply]
I mean one can refer to these values later in the markdown file, like `${page.title}` or `${page.date}`.

And thanks for the tip of using a page loader. Considering…
